pkgs/development/tools/capnproto-rust/default.nix

```diff
@@ -6,15 +6,20 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "capnproto-rust";
-  version = "0.17.1";
+  version = "0.17.2";
 
   src = fetchCrate {
     crateName = "capnpc";
     inherit version;
-    sha256 = "sha256-7RfJUYV3X9w0FALP3pbhmeIqrWLqlgr4oNvPnBc+RY8=";
+    sha256 = "sha256-WVjXVLVoTCAtA8a6+zaX4itAaPCWb2c0trtSsxBopO4=";
```

Comment:
Nit: This is just a personal preference, but since we are already using `cargoHash` (instead of `cargoSha256`) and SRI hashes everywhere, would it be possible to change `sha256 =` to `hash =`?

Reply:
Done.